DOCKER.md

```diff
@@ -58,6 +58,9 @@ Optionally, if you need special Jenkins configuration, you can mount JCasC YAML
 [Configuration-as-Code documentation](https://github.com/jenkinsci/configuration-as-code-plugin)
 for available options and the [JCasC demo](demo/casc/README.md) for an example.
 
+To get an interactive Jenkins CLI shell in the container, pass
+`-it -e FORCE_JENKINS_CLI=true` to `docker run` as extra parameters.
```

Comment:
Why do we need it for Docker?

Reply:
You mean the whole feature? I guess the same arguments apply as for normal mode, for debugging an existing installation, I don't see a reason not to support it. I'm not that familiar with all the use cases here though, so I can remove the docs (and the env var?) if desired